### PR TITLE
SUCCESS-67_Coordination of Benefits docs revamp

### PR DESCRIPTION
**Pull request by bstmartin**
addition of new parameters, a few changed

### CODE REVIEW

File: source/includes/_claims.md [hunk 1]
```diff
@@ -3749,7 +3749,7 @@ try client.claims(params: data)
 ```
 
 
-> Sample Claims request for sending COB (Coordination of Benefits) between two payers for Payer A.
+> Sample Claims request for sending COB (Coordination of Benefits) information for a subscriber between two payers for Payer A.
```
Comment (baldwindc) [inline source/includes/_claims.md]: what is language, "payer A" or "primary payer"? In the next snippet https://github.com/pokitdok/apidocs-v4/pull/214/files#diff-1ac5329d6bd94fd698b1a17d07c2abadR3992 we say primary.

File: source/includes/_claims.md [hunk 2]
```diff
@@ -4146,10 +4148,32 @@ client.claims({
           ],
           "date_paid": "2015-10-10",
           "remaining_patient_liability": "200"
+        },
+        {
+          "service_line_number": 2,
```
Comment (baldwindc) [inline source/includes/_claims.md]: I don't think we'll be sending service_line_number / line_level_adjustments to the primary payer, right?

Comment (bstmartin) [inline source/includes/_claims.md]: We won't, this is the example for COB sending to the secondary payer with adjustment information from the primary.

File: source/includes/_claims.md (hunk 1 above)
Comment (bstmartin) [inline source/includes/_claims.md]: Will edit this one to match the edit made on example two.

**Pull request merged**